src/layers/MultiscaleImageLayer/utils.js

```diff
@@ -47,6 +48,11 @@ export function renderSubLayers(props) {
     // Shared props with BitmapLayer:
     bounds,
     id: `tile-sub-layer-${bounds}-${id}`,
-    tileId: { x, y, z }
+    tileId: { x, y, z },
```

Comment:
Ah, I see ... Maybe we could add `INTERPOLATION_MODES.AUTO` that handles this behavior.

Reply:
I think you're right.  I was a little scared off by the idea that `AUTO` would be passed to layers where it does not make sense but we could just add checks there instead of having `null` indicate some sort of default.

Reply:
Ah the other hairy thing is that we moved away from "magic strings" so AUTO would have to be some sort of number while the others are integers:

```javascript
export enum INTERPOLATION_MODES {
  LINEAR = GL.LINEAR,
  NEAREST = GL.NEAREST
}
```

Not opposed to going back to strings, but I don't think we should mix WebGL constants and strings.  Any thoughts oon this @manzt ?

Reply:
https://www.typescriptlang.org/docs/handbook/enums.html#heterogeneous-enums

Reply:
Just opened a PR so the diff is clear - seems like a good move.